pkg/sql/table.go

```diff
@@ -240,7 +240,7 @@ func filterTableState(tableDesc *sqlbase.TableDescriptor) error {
 // getTableLease implements the SchemaAccessor interface.
 func (p *planner) getTableLease(tn *parser.TableName) (*sqlbase.TableDescriptor, error) {
 	if log.V(2) {
-		log.Infof(p.ctx(), "planner acquiring lease on table %s", tn)
+		log.Infof(p.ctx(), "planner acquiring lease on table %q", tn)
```

Comment:
%q produces a quoted string, so the output will look like

planner acquiring lease on table "test"